docs(devServer): add `writeToDisk` option (#2747)
webpack/webpack.js.org: src/content/configuration/dev-server.md

If this is too heavy on the file system, you can change this to an integer to set the interval in milliseconds.

See [WatchOptions](/configuration/watch/) for more options.

## `devServer.writeToDisk` 🔑

> `boolean` `function`

Reviewer: please add default value here and remove below. please add function signature here e.g. `function (param1, param2)`

Author: Done. thank you

> Default: `false`. Tell dev-server to write generated assets to the disk.

Reviewer: ```suggestion
Tells `devServer` to write generated assets to the disk.
```

__webpack.config.js__

```javascript
module.exports = {
  //...
  devServer: {
    writeToDisk: true
  }
};
```

> This option also accepts a Function value, which can be used to filter which files are written to disk. The function follows the same premise as [`Array#filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) in which a return value of false will not write the file, and a return value of true will write the file to disk.

Reviewer: ```suggestion
Providing a `Function` to `devServer.writeToDisk` can be used for filtering. The function follows the same premise as [`Array#filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) in which a boolean return value tells if the file should be written to disk.
```
